apps/web/src/components/Composer/NewPublication.tsx

```diff
@@ -64,8 +64,9 @@ import { PUBLICATION } from 'src/tracking';
 import type { LensterAttachment } from 'src/types';
 import { Button, Card, ErrorMessage, Spinner } from 'ui';
 import { v4 as uuid } from 'uuid';
-import { useContractWrite, useProvider, useSigner, useSignTypedData } from 'wagmi';
+import { useContractWrite, useProvider, useSignTypedData, useSigner } from 'wagmi';
```

Comment:
Is this changed by eslint?

Reply:
CI and Vercel fails

<img width="1243" alt="image" src="https://user-images.githubusercontent.com/69431456/230703801-1019a14d-a22d-4949-bb92-2b2aa99f7dd5.png">

Reply:
> Is this changed by eslint?

Yeah this was changed by eslint. I'll see if I can revert whatever changes it made

Reply:
> CI and Vercel fails
> 
> <img alt="image" width="1243" src="https://user-images.githubusercontent.com/69431456/230703801-1019a14d-a22d-4949-bb92-2b2aa99f7dd5.png">

:( Perhaps the eslint changes caused it to fail?